apps/src/pythonlab/pyodideWebWorker.js

```diff
@@ -2,7 +2,7 @@ import {loadPyodide} from 'pyodide';
 
 async function loadPyodideAndPackages() {
   self.pyodide = await loadPyodide({
-    indexURL: 'https://cdn.jsdelivr.net/pyodide/v0.24.1/full',
+    indexURL: '/assets/js/pyodide/',
```

Comment:
Would there be value versioning this on our end as well?

Reply:
The source of truth for the version is in [package.json](https://github.com/code-dot-org/code-dot-org/blob/9ca8eb33defacfb151e3d9fd0523a09badecd991/apps/package.json#L262). We don't need to put a version here because it will always be copied to the same folder.

Reply:
What about cache-busting? I believe we heavily cache `/assets/js` paths in cloudformation, and rely on sprockets adding md5 hashes to all the the filenames. Will that work here?

Reply:
Good call out. We won't be updating this frequently (maybe every few months to six months). I don't think we have control of the end of the url here (this is the prefix that the filenames will get appended to). So I think that means we can't cache-bust? Is there a way when we upgrade to force the cache to clear?

Reply:
Looks like we can even fetch the version programmatically! https://github.com/pyodide/pyodide/blob/fd80c7b1ed3ebfba4b136574b1e787984b2ef9fb/src/js/pyodide.ts#L20

Reply:
Ooh thanks! Got it working so it's versioned now!